CMakeLists.txt

```diff
@@ -20,6 +20,12 @@ set(hpxmp_headers
         src/kmp_atomic.h
         src/loop_schedule.h)
 
+set(SHOW_TRACE_DEBUG_MODE ON CACHE BOOL "display traces in debug mode")
```

Comment:
I'd suggest we introduce a naming convention for the HPXMP configuration constants. What about if we align it what HPX is doing: `HPXMP_WITH_FOO` for cmake and `HPXMP_HAVE_FOO` as the corresponding preprocessor constant?